### DISCUSSION TITLE
[Improvment] ShowPostNavLinks in reverse order

### DISCUSSION THREAD

**Discussion by alessiogatto** [adityatelange/hugo-PaperMod#826] [General]
Hi,
I think that could be a good idea to add a param (for example, reverseShowPostNavLinks: bool) that specify the order of the suggestion below a post.
It is possible to do this thing currently? For example, i want to reverse the "Next page" with the "Prev page" in all the posts. How can i do this?

Thanks

AG

**Comment by pushblue** [comment]
I wrote a little post on how to customize PaperMod to do just that:

https://push.blue/posts/20220718-reverse-papermodnavlinks/

**Reply by suiluj** [reply]
Thanks for your blog post with easy to follow instructions! And I checked your navigation links too. They work in the *correct* order. 😄

**Comment by Beryesa** [comment]
I also find it confusing, these could be either serial (current behaviour) or chronological like the usual (should revert)
It's easy to swap by hand but a small config option would be better instead of maintaining a set of patches for a set of patches of another theme :sweat_smile:

**Reply by tomtom94** [reply]
Yes I confirm it's confusing
